spec/spec_helper.rb

```diff
@@ -4,6 +4,7 @@
 require "vcr"
 
 SimpleCov.minimum_coverage 100
+SimpleCov.coverage_dir "public/coverage"
 SimpleCov.start "rails"
```

Comment:
Out of simplicity we could convert this three lines to:

```
SimpleCov.start("rails") do
  minimum_coverage 100
  coverage_dir "public/coverage"
end

Reply:
adapted